capabilities/istio/.github/zarf-runner/zarf.yaml

```diff
@@ -0,0 +1,25 @@
+kind: ZarfPackageConfig
```

Comment:
is this duplicate code from the other zarf runner folder in the root directory?

Reply:
the zarf runner steps are not generic so in the root it would be for the bundle. If we had one file ala task file or like what jordan had put together (wren) where i could call a specific task this wouldnt be a problem